kinetic/distribution.yaml

```diff
@@ -3687,6 +3687,21 @@ repositories:
       url: https://github.com/ros-interactive-manipulation/household_objects_database_msgs.git
       version: hydro-devel
     status: end-of-life
+  hr_msgs:
+    doc:
+      type: git
+      url: https://github.com/hansonrobotics/hr_msgs.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/hansonrobotics/hr_msgs.git
```

Comment:
This is your main repository not a release repository. This whole 'release' section should likely not be there if you're just looking to be documented and indexed. 

The release section is for making releases to be built into binary packages.

Reply:
Now I see. Thank you. Will remove it.